### PR TITLE
Improve device information when via device is unknown

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change
Improve device information when via device is unknown

Related backend PR which makes this less likely to happen in the future: https://github.com/home-assistant/core/pull/60153

Before, without this PR:
image

With this PR:
image

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -2316,10 +2316,10 @@
             "disable_error": "Enabling or disabling of the integration failed",
             "manuf": "by {manufacturer}",
             "hub": "Connected via",
```
Comment (bramkragten) [inline src/translations/en.json:2318]: This one is unused can be removed

**Review by bramkragten — DISMISSED** [review]

**Comment by balloob** [comment]
It now looks like "Device unavailable" applies to the device we're looking at, instead of describing the "via" device. 

As this is caused by a bug in the backend (not cleaning up via devices), I suggest we just leave it like this because no one should ever see it again.

**Comment by emontnemery** [comment]
> It now looks like "Device unavailable" applies to the device we're looking at, instead of describing the "via" device.

Right, hence this PR.

> this is caused by a bug in the backend (not cleaning up via devices), I suggest we just leave it like this because no one should ever see it again

The backend fix doesn't fix devices which already link to non-existing via-devices, I still think this PR should be merged?

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
Ah sorry I misunderstood the order of pictures. This is ok.

**Pull request merged**